eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/metrics/MetricInstrumentUnit.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.runtime.metrics;
+
+
+public abstract class MetricInstrumentUnit {
+
+    private MetricInstrumentUnit() {
+
+    }
```

Comment:
We can put this constant class into `constant` package together with `MonitorMetricConstants`.

>This comment is a copy of https://github.com/apache/eventmesh/pull/3446#discussion_r1451026815

Reply:
>Using an abstract class makes it easier for future extensions.

Did you originally mean to post this sentence in this conversation?

Reply:
This mainly consists of units for observability metrics. In my view, it's more appropriate to place them in a separate class, removing the `abstract` keyword from the class.